sentry_sdk/tracing_utils.py

```diff
@@ -180,6 +179,25 @@ def _get_frame_module_abs_path(frame):
         return None
 
 
+def _should_be_included(
+    is_sentry_sdk_frame: bool,
+    namespace: Optional[str],
+    in_app_include: Optional[List[str]],
+    in_app_exclude: Optional[List[str]],
+    abs_path: Optional[str],
+    project_root: Optional[str],
+) -> bool:
```

Comment:
We are still using type comments in most places. I will change these to type comments before merge to ensure consistency with the rest of the code